railgun/test/integration/suite_test.go

```diff
@@ -81,6 +82,12 @@ var (
 	// dbmode indicates the database backend of the test cluster ("off" and "postgres" are supported)
 	dbmode = os.Getenv("TEST_DATABASE_MODE")
 
+	// ClusterVersionEnvVar indicates the environment variable used to configure the Kubernetes cluster version.
+	ClusterVersionEnvVar = "KONG_CLUSTER_VERSION"
+
+	// clusterVersion indicates the version of Kubernetes to use for the tests (if the cluster was not provided by the caller)
+	clusterVersionStr = os.Getenv(ClusterVersionEnvVar)
+
```

Comment:
style inconsistent with the variable `dbmode` above.

The indirection with `ClusterVersionEnvVar` seems superfluous to me but that's only my preference. Also, the env var name appears unnecesarily exported.

Reply:
Sounds good, let's tear it out:

```suggestion
	// clusterVersion indicates the version of Kubernetes to use for the tests (if the cluster was not provided by the caller)
	clusterVersionStr = os.Getenv("KONG_CLUSTER_VERSION")

```